kinto_wizard/yaml2kinto.py

```diff
@@ -142,6 +144,25 @@
                                                 collection=collection_id,
                                                 data=record_data,
                                                 permissions=record_permissions)
+                if delete_missing_records and collection_records and \
+                   'records' in current_collection:
+                    # Fetch all records IDs
+                    file_records_ids = set(collection_records.keys())
+                    server_records_ids = set(current_collection['records'].keys())
+
+                    to_delete = server_records_ids - file_records_ids
+                    if not force:
+                        message = ("Are you sure that you want to delete the "
+                                   "following {} records?".format(len(list(to_delete))))
+                        value = input(message)
```

Comment:
I don't remember but we may have an option to always accept prompts, no?

Reply:
We currently don't have any prompts. We might want to add a ``-y --yes`` option though.